README.rst

```diff
@@ -0,0 +1,91 @@
+LTI Consumer XBlock |Build Status| |Coveralls|
+----------------------------------------------
+
+This XBlock implements the consumer side of the LTI specification enabling
+integration of third-party LTI provider tools.
+
+Installation
+------------
+
+Install the requirements into the python virtual environment of your
+``edx-platform`` installation by running the following command from the
+root folder:
+
+.. code:: bash
+
+    $ pip install -r requirements.txt
+
+Enabling in Studio
+------------------
+
+You can enable the LTI Consumer XBlock in Studio through the
+advanced settings.
+
+1. From the main page of a specific course, navigate to
+   ``Settings ->    Advanced Settings`` from the top menu.
+2. Check for the ``advanced_modules`` policy key, and add
+   ``"lti_consumer"`` to the policy value list.
+3. Click the "Save changes" button.
+
+Workbench installation and settings
+-----------------------------------
+
+Install to the workbench's virtualenv by running the following commands
+from the xblock-lti-consumer repo root:
+
+.. code:: bash
+
+    $ pip install -r requirements.txt
+    $ pip install -r test_requirements.txt
+    $ mkdir var
+
+Running the workbench
+---------------------
+
+Run the workbench from the xblock-sdk repo root:
+
+.. code:: bash
+
+    $ ./manage.py runserver 8000
+
+Access it at `http://localhost:8000/ <http://localhost:8000>`__.
+
+Running tests
+-------------
+
+From the lti_consumer directory, run the tests with the following command:
+
+.. code:: bash
+
+    $ DJANGO_SETTINGS_MODULE="workbench.settings" nosetests tests/*
```

Comment:
This appears to conflict slightly with what is contained in the .travis.yml file.